californium-proxy/src/test/java/org/eclipse/californium/proxy/HttpTranslatorTest.java

```diff
@@ -33,7 +33,7 @@
 
 	@Test
 	public void testGetHttpEntity() throws Exception {
-		Request req = new Request(Code.GET);
+		Request req = new Request(Code.PUT);
```

Comment:
I maybe don't get the point here, but should we rename the test name too?

Reply:
the next line calls `req.setPayload("payload");` so I changed it.
Yes, the test should be then renamed.